crates/lib/src/lib.rs

```diff
@@ -41,7 +41,7 @@ pub use type_value::{AlgebraicValue, ProductValue};
 
 pub use spacetimedb_sats as sats;
 
-pub const MODULE_ABI_VERSION: VersionTuple = VersionTuple::new(3, 0);
```

Comment:
This should should check one of the "breaking change" boxes in the PR template, I'd assume.

Reply:
Yeah I'd think so, but then it's weird that we don't have *ABI* breaking checkbox, only *API* ones. This PR doesn't break any of the public APIs, but it does break ABI.

Reply:
https://github.com/clockworklabs/SpacetimeDB/pull/213 :) next time

Reply:
Thanks, approved!